### ISSUE TITLE
Consider adding options to run.sh 

### ISSUE THREAD

**Issue by ghost** [scala/community-builds#474]
Currently, the main way to override value set in `run.sh` is to overrwrite local files. If options were added, not only is this cleaner for the obvious reasons, but would also allow standard variations to be kept in the repository - e.g. one config for latest (master), one for stable, etc 

Suggested changes:

- Create directory `configs/scala` and move `community.dbuild, project-refs.conf, resolvers.conf ` there.

- Modify run.sh to take options, passing all other options to dbuild (eg -l)
- Add options (--cb to highlight community build options, not standard dbuild ones);
```
--cb-config         base directory for configurations:  default `configs/scala`
--cb-config-file    dbuild file: default $cb-config/community.dbuild
--cb-project-refs   project-refs file: default $cb-config/project-refs.conf
--cb-resolvers      resolvers file: default $cb-config/resolvers.conf
--cb-version        scala version, currently set via env: default as now (maybe even use env var, if set)
```
- Modify current `community.dbuild` to get new file values from the evironment

These changes should allow local overrides (without modifying the originals) and also allow
new configs to be put in subdirectories (tls_scala, typelevel, scala.js) in the main repo directory,
or completely separate

**Comment by ghost** [comment]
First cut (but working) at https://github.com/BennyHill/community-builds/tree/fix/474

**Comment by SethTisue** [comment]
fixed by #478
